src/style.css

```
@@ -1,3 +1,67 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
```

Use width 100 instead
```suggestion
  justify-content: space-between;
  width: 100%;
  padding: 0 50px;
  box-sizing: border-box;
```